pkg/minishift/docker/image/sysproc_linux.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"os"
+	"syscall"
+)
+
+func sysProcForBackgroundProcess() *syscall.SysProcAttr {
+	sysProcAttr := new(syscall.SysProcAttr)
+	sysProcAttr.Setpgid = true
+	sysProcAttr.Pgid = 0
```

Comment:
For the scenario you mentioned it should be fine to use parent's process group as children can continue even if the leader of the process group dies. Creating a new process group is giving the children more autonomy (in some way). So was concerned if there is any downside to this. Will do little testing around this. But most likely the current implementation is fine.